demos/demo_resolve_conflicts.sh

```diff
@@ -16,21 +16,20 @@ run_command "jj describe -m \"README: say which world\""
 run_command "echo \"Hello Earth!\" > README"
 run_command "jj diff"
 
-comment "We're going to rebase it onto commit b1.
+comment "We're going to rebase it onto commit b1b.
```

Comment:
It doesn't seem like this needed to change... Hmm, I guess it depends on the timestamp, so it varies from run to run. But most of the time it shouldn't be needed, so let's leave it unchanged, or what do you think?

Reply:
I ran into errors from more than one b1 commit a few times. It was a bit annoying to have to be on the lookout for it.

Reply:
It would be nice if we can make the IDs deterministic like we do in tests, but that doesn't have to be part of this PR. I'm just worried it's be distracting for the reader if we sometimes use a longer prefix than what's displayed in the screenshots

Reply:
I'll add a comment. I think if we go this far, we should stop using shell scripts and set up a [cargo xtask](https://github.com/matklad/cargo-xtask) using our testing library and one of [these libraries](https://github.com/matklad/cargo-xtask?tab=readme-ov-file#tooling).

If you think this is distracting, I'll revert it to two letters. It makes the job of making demos slightly more annoying, but it shouldn't happen too often. I might wait until the last moment to make this change, though.

Reply:
I also realized there's *another* place where `b1` is used. On one hand, that's probably why I saw errors relatively often. More importantly, I should make sure the two places are consistent.

Reply:
We could switch this to use something like `jj log -T 'commit_id.shortest()' --color=never`. I might do that or just add a TODO.